Serializer/Jms/Handler/ResourceHandler.php

```diff
@@ -86,16 +86,15 @@ private function doSerializeResource(Resource $resource, $depth = 0)
 
         $data['path'] = $resource->getPath();
         $data['repository_path'] = $resource->getRepositoryPath();
-        $children = $resource->listChildren();
 
         $enhancers = $this->enhancerRegistry->getEnhancers($repositoryAlias);
 
         $children = array();
-        foreach ($resource->listChildren() as $name => $resource) {
+        foreach ($resource->listChildren() as $name => $childResource) {
             $children[$name] = array();
 
-            if ($depth < 1) {
-                $children[$name] = $this->doSerializeResource($resource, $depth + 1);
+            if ($depth < 2) {
```

Comment:
Hmm, for the TreeBrowserBundle, it has to be at least a depth of 2 (otherwise, only root + cms node will be rendered at first). What about defaulting to 2 for now and creating a PR to make it configurable (so that TreeBrowser can prepend it)